scipy/stats/_discrete_distns.py

```diff
@@ -344,11 +344,11 @@ def _logcdf(self, x, n, p):
         def f1(k, n, p):
             return np.log1p(-special.betainc(k + 1, n, 1 - p))
 
-        def f2(k, n, p):
-            return np.log(cdf)
-
+        logcdf = np.empty_like(cdf)
```

Comment:
```suggestion
        logcdf = cdf
```
This would be a little more efficient.

Reply:
I agree. Feel free to commit this.